File: backend/judicial-warrant/src/main/java/com/informatique/gov/judicialwarrant/rest/request/CapacityDelegationChangeStatusRequest.java
package com.informatique.gov.judicialwarrant.rest.request;

import java.io.Serializable;

import com.informatique.gov.judicialwarrant.rest.dto.CapacityDelegationDto;

import lombok.Data;


@Data
public class CapacityDelegationChangeStatusRequest implements Serializable{

	/**
	 * 
	 */
	private static final long serialVersionUID = -6698575712022936099L;

	private CapacityDelegationDto capacityDelegation;
	private String note;
}

File: frontend/jwapp/src/models/CapacityDelegationChangeStatusRequest.js
module.exports = function(app){
    app.factory('CapacityDelegationChangeStatusRequest', function(CapacityDelegation){
        return function CapacityDelegationChangeStatusRequest(){
            var self = this;
            self.capacityDelegation = new CapacityDelegation();
            self.note = null;
            self.version = 0;
            
            
        }
    });
};